lib/tests/streamlit/elements/media_test.py

```diff
@@ -26,29 +26,29 @@
 from tests.delta_generator_test_case import DeltaGeneratorTestCase
 
 
-class TestMediaKind(Enum):
+class MockMediaKind(Enum):
```

Comment:
pytest tries to "collect" this enum class to run tests on, because the class name matches "Test*". Renamed to avoid warnings.